src/commonTest/kotlin/fr/acinq/lightning/crypto/LocalKeyManagerTestsCommon.kt

```diff
@@ -36,6 +37,29 @@ class LocalKeyManagerTestsCommon : LightningTestSuite() {
         assertEquals(DeterministicWallet.encode(keyManager.revocationPoint(keyPath), DeterministicWallet.tpub), "tpubDKeRVNEjuhXHK3dHA96zJnWc7i2NM5Hr1HWMVQyo9VuV1tj3GGZCrJkcDa9f1wWamEikqFYqh4xdCs9HeNc6HwPy6YtmVacpCfQpAT8PaG5")
     }
 
+    @Test
+    fun `generate channel keys`() {
+        val seed = ByteVector("aeb3e9b5642cd4523e9e09164047f60adb413633549c3c6189192921311894d501")
+        val keyManager = LocalKeyManager(seed, Block.TestnetGenesisBlock.hash)
+        val fundingKeyPath = makefundingKeyPath(ByteVector("06535806c1aa73971ec4877a5e2e684fa636136c073810f190b63eefc58ca488"), isFunder = false)
+        val channelKeys = keyManager.channelKeys(fundingKeyPath)
+
+        // README !
+        // test data generated with v1.0-beta11, but they should never change
+        // if this test fails it means that we cannot restore channels created with older phoenix of lightning-kmp without
```

Comment:
```suggestion
        // if this test fails it means that we cannot restore channels created with older versions of lightning-kmp without
```

Reply:
done in https://github.com/ACINQ/lightning-kmp/pull/255/commits/3b01af57bbb8bc95af625ce1d11ae54e433214de